lib/MusicBrainz/Server/Data/EditNote.pm

```diff
@@ -132,6 +132,66 @@ sub find_by_recipient {
     );
 }
 
+sub remove_content {
+    my ($self, $edit_note, $changing_editor_id, $reason) = @_;
+
+    my $edit_note_id = $edit_note->{id};
+    my $edit_note_text = $edit_note->{text};
+    my @args = (
+        $edit_note_id,
+        $changing_editor_id,
+        $edit_note_text,
+        $reason,
+        $edit_note_id
+    );
+
+    $self->sql->do(<<~"SQL", @args);
+        INSERT INTO edit_note_change (
+                        status, edit_note, changing_editor,
+                        old_note, new_note, reason
+                    )
+             VALUES (
+                        'deleted', ?, ?,
+                        ?, '', ?
+                    );
+
+        UPDATE edit_note
+           SET text = ''
+         WHERE id = ?;
+        SQL
+}
+
+sub edit_content {
+    my ($self, $edit_note, $changing_editor_id, $new_content, $reason) = @_;
+
+    my $edit_note_id = $edit_note->{id};
+    my $edit_note_text = $edit_note->{text};
+    my @args = (
+        $edit_note_id,
+        $changing_editor_id,
+        $edit_note_text,
+        $new_content,
+        $reason,
+        $new_content,
+        $edit_note_id
+    );
+
+    $self->sql->do(<<~"SQL", @args);
+        INSERT INTO edit_note_change (
+                        status, edit_note, changing_editor,
+                        old_note, new_note, reason
+                    )
+             VALUES (
+                        'edited', ?, ?,
+                        ?, ?, ?
+                    );
```

Comment:
The spacing/indentation here seems kind of excessive somehow :)  Is this what the WIP SQL style guide we had suggested?

Reply:
Not exactly, but that might be worse still - it says "Subqueries and other parentheses-enclosed blocks should be aligned as though the open parenthesis were the 0-column. So, they should be indented as a unit, to identify them as a block. Subqueries should continue to have the opening keywords right-aligned." so `status` should be further right than the parens in `edit_note_change`...

Reply:
I don’t find it excessive. I wouldn’t mind aligning on parenthesis’ next column either, but it’s fine to me as-is for now.

Reply:
Fine as-is for me too.